Add `sma_ennexos` brand assets

shadow578 opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).
-->
## Proposed change
<!-- 
  Describe the big picture of your changes here to communicate to the
  maintainers why we should accept this pull request.
-->

Adds brand assets for `sma_ennexos` custom integration for integrating with [ennexOS](https://www.sma.de/produkte/apps-software/ennexos) devices.
All assets are derived from SVG base images.

## Type of change
<!--
  What type of change does your PR introduce to the Home Assistant Brands?
  NOTE: Please, check only 1! box! 
  If your PR requires multiple boxes to be checked, you'll most likely need to
  split it into multiple PRs. This makes things easier and faster to code review.
-->

- [ ] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [X] Add a new logo or icon for a custom integration (custom component)
  - [X] I've added a link to my custom integration repository in the PR description
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Replace an existing icon or logo after a branding change
- [ ] Removing an icon or logo

## Additional information
<!--
  Details are important, and help maintainers processing your PR.
  Please be sure to fill out additional details, if applicable.
-->

- This PR fixes or closes issue: fixes #
- Link to code base pull request: 
- Link to documentation pull request: 
- Link to integration documentation on our website: 
- Link to custom integration repository: https://github.com/shadow578/homeassistant_sma-ennexos

## Checklist
<!--
  Put an `x` in the boxes that apply. You can also fill these out after
  creating the PR. If you're unsure about any of them, don't hesitate to ask.
  We're here to help! This is simply a reminder of what we are going to look
  for before merging your contribution.
-->

- [X] The added/replaced image(s) are **PNG**
- [X] Icon image size is 256x256px (`icon.png`)
- [X] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [X] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [X] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

<!--
  Thank you for contributing <3
-->

frenck reviewed (changes requested):
Hi there @shadow578 👋 

Thanks for opening up a pull request. However, the provided logos assets seems to be product branding, not manufacturer branding. Please adjust this PR to use the manufacturer branding instead.

Thanks! 👍 

../Frenck

shadow578 commented:
Hi @frenck ,

thanks for the feedback. 
I wasn't aware of the requirement that logos must be manufacturer branding.

Anyway, i've now updated the logo to use the general branding used by SMA Solar AG.
Additionally, i've removed a slight border on the icons that i didn't notice previously.

frenck reviewed (approved):
Nice! Thanks, @shadow578 👍 

../Frenck
